packages/core/src/Stores/Modules/Cashier/on-ramp-store.js

```diff
@@ -196,14 +257,24 @@ export default class OnRampStore extends BaseStore {
         this.is_deposit_address_loading = is_loading;
     }
 
+    @action.bound
+    setIsDepositAddressPopoverOpen(is_open) {
+        this.is_deposit_address_popover_open = is_open;
+    }
+
     @action.bound
     setIsOnRampModalOpen(is_open) {
         this.is_onramp_modal_open = is_open;
     }
 
     @action.bound
-    setIsDepositAddressPopoverOpen(is_open) {
-        this.is_deposit_address_popover_open = is_open;
+    setIsRequestingWidgetHtml(is_requesting_widget_html) {
+        this.is_requesting_widget_html = is_requesting_widget_html;
+    }
+
+    @action.bound
+    setIsWidgetHtmlLoading(is_widget_html_loading) {
```

Comment:
Could this function + observable be removed? I don't see the function called anywhere 🤔

Reply:
Definitely, this is a remnant of an old version. 👍